db/seeds.rb

```diff
@@ -6,14 +6,15 @@
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
 role = Role.find_or_create_by!(name: 'admin')
-user = User.find_or_create_by!(email: "test@gmail.com", password: "password", role_id: role.id)
+user = User.first || User.find_or_create_by!(email: "test@gmail.com", password: "password", role_id: role.id)
```

Comment:
Kindly when we seed the db it throws this error. It seems the || doesn't work as expected. it renders an error that says there is no column password for users. 
![Screenshot from 2022-11-15 13-16-26](https://user-images.githubusercontent.com/77842237/201880304-8f911f23-f4ec-4861-847c-17f0c59c083a.png)

Reply:
@Tresor11  Could you fix the bug, please? we're blocked because of that.

Reply:
@vic778 I think it would we better to open a PR based on main to fix that error then we would rebase the branch
That would allow us to keep the scope of this PR